xray/plot/plot.py

```diff
@@ -452,6 +452,8 @@ def newplotfunc(darray, ax=None, xincrease=None, yincrease=None,
             # pcolormesh
             kwargs['extend'] = cmap_params['extend']
             kwargs['levels'] = cmap_params['levels']
+            if 'colors' in kwargs:
```

Comment:
`colors` and `cmap` should probably be required to be mutually exclusive? I don't think it makes sense to supply both.